builtin/providers/aws/resource_aws_ebs_volume_test.go

```diff
@@ -1,23 +1,59 @@
 package aws
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/service/ec2"
 	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
 )
 
 func TestAccAWSEBSVolume(t *testing.T) {
+	var v ec2.Volume
 	resource.Test(t, resource.TestCase{
 		PreCheck:  func() { testAccPreCheck(t) },
 		Providers: testAccProviders,
 		Steps: []resource.TestStep{
 			resource.TestStep{
 				Config: testAccAwsEbsVolumeConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckVolumeExists("aws_ebs_volume.test", &v),
+				),
 			},
 		},
 	})
 }
 
+func testAccCheckVolumeExists(n string, v *ec2.Volume) resource.TestCheckFunc {
```

Comment:
Upgraded this test while I was here, since I needed a `testAccCheckVolumeExists` method